pages/cart.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cart | CrickStock</title>
+    <link rel="shortcut icon" href="../assets/favicon.jpeg" type="image/x-icon">
+    <link rel="stylesheet" href="https://tarang-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../stylesheets/base.css">
+    <link rel="stylesheet" href="../styesheets/base.css">
+</head>
+<body>
+    <!-- Header will be inserted dynamically  -->
+    <main class="pd-md">
+        <h2 class="txt-center">My cart</h2>
+        <div class="d-flex f-wrap justify-c-center align-i-start gap-xlg pd-y-lg">
+            <div id="productCardsContainer" class="cart-items flex-col gap-md">
+                <!-- Cart Items to be added dynamically  -->
+            </div>
+            <aside class="cart-summary tr-card flex-col gap-md pd-md">
+                <h3 class="txt-semibold">Price details</h3>
+                <hr>
+                <div class="flex-col">
+                    <div class="d-flex justify-c-space-between">
+                        <p>Price (2 items)</p> <p>₹4,000</p>
```

Comment:
suggestion: you can also use a small or span element for small text! p tag is also good

Reply:
Thank you for pointing it out, I have changed it in the latest commit